### PR TITLE
C1 abdelrahman edit post

### PR DESCRIPTION
**Pull request by abdelrahmanMaged**
@Reem-Sharkawy @Tharwat7 @BeshoSmile 
PLease review :D

### CODE REVIEW

**Comment by abdelrahmanMaged** [comment]
@Tharwat7 @BeshoSmile @Reem-Sharkawy it is a kind reminder to review the branch.
Regards,

**Comment by abdelrahmanMaged** [comment]
@Tharwat7 @Reem-Sharkawy @BeshoSmile would you please review the branch :)

**Comment by Tharwat7** [comment]
@abdelrahmanMaged please check what you have done in the models.py since the interested_in method is Besho's

File: project/tager/tager_www/models.py [hunk 1]
```diff
@@ -229,6 +229,16 @@ def add_to_wish_list(self, post_id):
         else:
             return "true"
 
+    #The Method Takes 2 arguments(User who clicked intrested,Post Which the user has clicked the button in)
```
Comment (BeshoyAtef) [inline project/tager/tager_www/models.py:232]: ??

**Comment by abdelrahmanMaged** [comment]
@Reem-Sharkawy  please review the scenario :D

**Comment by abdelrahmanMaged** [comment]
@HebaElSobky please review

**Comment by HebaElSobky** [comment]
abdelrahman make sure the text box expands 3shan shaklo kda mela5bat

**Comment by HebaElSobky** [comment]
@AbdelrahmanAyad price says can not start with a zero, i just edited it and entered 00 w it was successfully updated, kindly check this exception again... sometimes it accepts 00 and sometimes it doesnt

**Comment by HebaElSobky** [comment]
@AbdelrahmanAyad 
1)after editing the title the ok stills appears please make sure it's removed after editing is done 
2) after editing if i press back and the go back to the editing page, the edited info is not saved.. it returns back to the values it was created with

**Comment by HebaElSobky** [comment]
price 00 ??

**Comment by abdelrahmanMaged** [comment]
i will fix all the bugs but regarding the back button it is something
related to the caching in the browser

**Comment by HebaElSobky** [comment]
00?

**Comment by abdelrahmanMaged** [comment]
i will push in 10 minutes all the remarks can you please wait those 10 mins
and complete the reviewing ?? :D

**Comment by abdelrahmanMaged** [comment]
@HebaElSobky all you comments has been taken in consideration :) regards :D

**Comment by Tharwat7** [comment]
@abdelrahmanMaged 
Please remove the <<<<<master and heads in the files

**Comment by abdelrahmanMaged** [comment]
i removed them :O

**Comment by abdelrahmanMaged** [comment]
@Tharwat7  i am sorry i will remove them now

**Comment by HalaKamal** [comment]
@AbdelrahmanAyad  tage me when u finish the diagram

**Comment by HalaKamal** [comment]
@AbdelrahmanAyad  your diagram has some defaults 
1- the arrows that are requesting and rendering aren't clear 
2- the viewlist.html must be in templates not in tager
the sequeb=nce uml diagram is reopended

**Comment by HalaKamal** [comment]
@AbdelrahmanAyad @abdelrahmanMaged  by the way i'm your reviewer of uml diagram i'm not supposed to look after you that you uploaded your things to review it u must tag me 
regards

**Pull request merged**